### ISSUE TITLE
Black Screen

### ISSUE THREAD

**Issue by void-dreemer** [noryb009/lick#57]
Hello all,

I tried searching to see if anyone has had this issue but I'm not having any luck... I used LICK to install fossapup64-9.5 it installed fine, upon reboot I can choose Windows or puppy Linux just fine, however, once I choose puppy Linux it stays at a black screen with a cursor "_" (similar to a command prompt underscore); that's all I see, nothing else.

Has anyone had this issue? Can anyone point me in the right direction?

I'm on a ideapad gaming laptop (Lenovo) with an AMD Ryzen 5 and an NVidia GEFORCE GTX GPU (not sure if that helps at all). I disable and enabled secure boot and it still happens.

**Comment by noryb009** [comment]
Have you tried reinstalling the boot loader? If any of the files are corrupt, then this should fix the issue.

If that doesn't help, were you able to [run these instructions](https://github.com/noryb009/lick#uefi-systems-with-secure-boot) at some point in time? If not, then the shim might be having trouble with your system, and we can try to remove that. If so, then the issue is likely with grub, and we can try to change the configuration around.

**Comment by void-dreemer** [comment]
> 

Hello @noryb009,

Thank you for your quick reply! Yes, I reinstalled the boot loader and I did try running those instructions; at first I didn't even get the blue screen or anything, then I realized that I had secure boot disabled (even though I previously "enabled" it, there was an issue that wasn't allowing it to enable, and I fixed that problem), finally got secure boot enabled, got the blue screen and did as the instructions stated. However, after hitting exit, and trying to boot back into puppy Linux, I got the same black screen with a cursor and nothing else. 

I'm at a loss, I love LICK and feel like there really isn't any other tool I like more. So I'm hoping this can somehow be resolved. I'll try a few more things and keep you updated! I feel like the issue is grub (as you previously stated before).
